packer/app.json

```
@@ -0,0 +1,76 @@
+{
+
```

Нужно отрефакторить файл, уж очень много пустых строк.

---

Добрый день! "Отрефакторил" и запушил. 

```suggestion
{  
   "builders":[  
      {  
         "type":"googlecompute",
         "project_id":"{{ user `project_id` }}",
         "image_name":"reddit-app-base-{{timestamp}}",
         "image_family":"reddit-app-base",
         "source_image_family":"ubuntu-1604-lts",
         "ssh_username":"{{ user `ssh_username` }}",
         "machine_type":"{{ user `machine_type` }}",
         "zone":"{{ user `zone` }}"
      }
   ],
   "provisioners":[  
      {  
         "type":"shell",
         "script":"scripts/install_ruby.sh",
         "execute_command":"sudo {{.Path}}"
      }
   ]
}
```